components-java-sdk/src/test/java/io/dapr/components/wrappers/OutputBindingGrpcComponentWrapperTest.java

```diff
@@ -30,6 +30,7 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
+
 class OutputBindingGrpcComponentWrapperTest {
```

Comment:
Is the extra line required?

Reply:
Not really. Removing (I thought we required 2 blank lines before a class definition but that might be a python thing instead :P)

Reply:
Fixed on 827915b7b19570223ffd5d94757be41a6947c545